src/components/AutoCompleteInput/AutoCompleteInput.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import Downshift from 'downshift';
+import { includes } from 'lodash/fp';
+
+import SearchInput from '../SearchInput';
+import Card from '../Card';
+import { textMega } from '../../styles/style-helpers';
+
+const AutoCompleteWrapper = styled('div')`
+  label: input__container
+  position: relative;
+  min-width: 150px;
+`;
+
+const ItemsWrapper = styled('div')`
+  position: relative;
+  height: 0px;
+  overflow: visible;
+  margin-top: ${props => props.theme.spacings.bit};
+`;
+
+const itemsBaseStyles = ({ theme }) => css`
+  padding: ${theme.spacings.kilo} ${theme.spacings.mega};
+  min-width: initial !important;
```

Comment:
😢

Reply:
I am sorry, I know this hurts, but for some reason I thought I was rebased on your branch, but in fact, I was not. Removed this.